rites/roman1969/src/locales/en.ts

```diff
@@ -938,6 +938,8 @@ export const locale: Locale = {
     peter_liu_wenyuan_martyr: 'Saint Peter Liu Wenyuan, Martyr',
     peter_nolasco_religious: 'Saint Peter Nolasco, Religious',
     peter_of_alcantara_priest: 'Saint Peter of Alcántara, Priest',
+    peter_poveda_and_innocent_of_mary_immaculate_canoura_priests_and_companions_martyrs:
+      'Saints Peter Poveda Castroverde and Innocent of Mary Immaculate Canoura Arnau, Priests, and Companions Martyrs',
```

Comment:
@TobiTenno, should we prodie the English Wikipedia articles aa the `based on` source of this? It is not accurate, as we have localised the names, but I don’t think we have better sources, unless the US Roman Missal also includes the celebration for Spaniah-speaking parts of the US.

Reply:
```suggestion
      // src: https://en.wikipedia.org/wiki/Pedro_Poveda_Castroverde, https://en.wikipedia.org/wiki/Inocencio_of_Mary_Immaculate, 
      'Saints Peter Poveda Castroverde and Innocent of Mary Immaculate Canoura Arnau, Priests, and Companions Martyrs',
```